src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -517,6 +532,154 @@ namespace AirflowNetworkBalanceManager {
+		if ( AirflowNetworkNumOfOccuVentCtrls > 0 ) {
+			OccupantVentilationControl.allocate( AirflowNetworkNumOfOccuVentCtrls );
+			for ( i = 1; i <= AirflowNetworkNumOfOccuVentCtrls; ++i ) {
+				GetObjectItem( CurrentModuleObject, i, Alphas, NumAlphas, Numbers, NumNumbers, IOStatus, lNumericBlanks, lAlphaBlanks, cAlphaFields, cNumericFields );
+				IsNotOK = false;
+				IsBlank = false;
+				VerifyName( Alphas( 1 ), OccupantVentilationControl.Name( ), i - 1, IsNotOK, IsBlank, CurrentModuleObject + " Name" );
+				if ( IsNotOK ) {
+					ErrorsFound = true;
+					if ( IsBlank ) Alphas( 1 ) = "xxxxx";
+				}
+
+				OccupantVentilationControl( i ).Name = Alphas( 1 ); // Name of object
+				OccupantVentilationControl( i ).MinOpeningTime = Numbers( 1 );
+				if ( OccupantVentilationControl( i ).MinOpeningTime < 0.0 ) {
+					ShowWarningError( RoutineName + CurrentModuleObject + " object, " + cNumericFields( 1 ) + " < 0.0" );
+					ShowContinueError( "..Input value = " + RoundSigDigits( OccupantVentilationControl( i ).MinOpeningTime, 1 ) + ", Value will be reset to 0.0" );
+					ShowContinueError( "..for " + cAlphaFields( 1 ) + " = \"" + OccupantVentilationControl( i ).Name );
+					OccupantVentilationControl( i ).MinOpeningTime = 0.0;
+				}
+				OccupantVentilationControl( i ).MinClosingTime = Numbers( 2 );
+				if ( OccupantVentilationControl( i ).MinClosingTime < 0.0 ) {
+					ShowWarningError( RoutineName + CurrentModuleObject + " object, " + cNumericFields( 2 ) + " < 0.0" );
+					ShowContinueError( "..Input value = " + RoundSigDigits( OccupantVentilationControl( i ).MinClosingTime, 1 ) + ", Value will be reset to 0.0" );
+					ShowContinueError( "..for " + cAlphaFields( 1 ) + " = \"" + OccupantVentilationControl( i ).Name );
+					OccupantVentilationControl( i ).MinClosingTime = 0.0;
+				}
+				if ( NumAlphas == 1 && NumNumbers == 2 ) {
+					OccupantVentilationControl( i ).MinTimeControlOnly = true;
+				}
+				if ( !lAlphaBlanks( 2 ) ) {
+					OccupantVentilationControl( i ).ComfortLowTempCurveName = Alphas( 2 );
+					OccupantVentilationControl( i ).ComfortLowTempCurveNum = GetCurveIndex( Alphas( 2 ) ); // convert curve name to number
+					if ( OccupantVentilationControl( i ).ComfortLowTempCurveNum == 0 ) {
+						OccupantVentilationControl( i ).MinTimeControlOnly = true;
+						ShowWarningError( RoutineName + CurrentModuleObject + " object, " + cAlphaFields( 2 ) + " not found = " + OccupantVentilationControl( i ).ComfortLowTempCurveName );
+						ShowContinueError( "..for specified " + cAlphaFields( 1 ) + " = " + Alphas( 1 ) );
+						ShowContinueError( "Thermal comfort will not be performed and minimum opening and closing times are checked only. Simulation continues." );
+					}
+					else {
+						// Verify Curve Object, only legal type is BiQuadratic
+						{ auto const SELECT_CASE_var( GetCurveType( OccupantVentilationControl( i ).ComfortLowTempCurveNum ) );
+						if ( SELECT_CASE_var == "LINEAR" ) {
```

Comment:
Yes. It should be OR instead of AND. Here is my suggestion:
if(SELECT_CASE_var != "LINEAR" || SELECT_CASE_var != "QUADRATIC")
@jasondegraw What do you think?